supertokens_python/recipe/session/interfaces.py

```diff
@@ -208,13 +216,21 @@ async def revoke_session(
 
     @abstractmethod
     async def revoke_all_sessions_for_user(
-        self, user_id: str, tenant_id: str, revoke_across_all_tenants: Optional[bool], user_context: Dict[str, Any]
+        self,
+        user_id: str,
+        tenant_id: str,
+        revoke_across_all_tenants: Optional[bool],
```

Comment:
this should not be optional in recipe interface

Reply:
Done